package.json

```diff
@@ -4,6 +4,9 @@
   "description": "JavaScript sprintf implementation",
   "author": "Alexandru Mărășteanu <hello@alexei.ro>",
   "main": "src/sprintf.js",
+  "files": [
+    "dist"
```

Comment:
I guess this should include a bit more like at least `LICENSE` and maybe `README.md` or `CHANGLOG`?

Reply:
[Source](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files)

```
Certain files are always included, regardless of settings:
    package.json
    README
    CHANGES / CHANGELOG / HISTORY
    LICENSE / LICENCE
    NOTICE
    The file in the "main" field
README, CHANGES, LICENSE & NOTICE can have any case and extension.
```
So, this is unnecessary.